package.json

```diff
@@ -102,7 +102,7 @@
     "postinstall": "node packageScripts/postinstall.js",
     "postinstall:lit": "node scripts/postinstall.js",
     "ciBuild": "npm-run-all sassBuild sassRender cssLint distJS bundler postinstall apiBuild",
-    "cssLint": "stylelint './src/*.css'",
+    "cssLint": "stylelint ./src/*.css",
```

Comment:
This modification allows the command to run without error on my Windows machine, which is a hook in the commit process.

Reply:
Please submit this to the WC-Generator repo as well! this is a great find.

Reply:
This is how we have this in the current version of the Generator. 
https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json#L101

When we ran into this in the past, that was the preferred solution.